postgresql/provider.go

```diff
@@ -60,6 +60,12 @@ func Provider() terraform.ResourceProvider {
 				Optional:   true,
 				Deprecated: "Rename PostgreSQL provider `ssl_mode` attribute to `sslmode`",
 			},
+			"connection_string": {
+				Type:        schema.TypeString,
+				Optional:    true,
+				DefaultFunc: schema.EnvDefaultFunc("PGCONNSTRING", nil),
```

Comment:
I'm not seeing this environment variable exposed anywhere via [`pq(3)`](https://www.postgresql.org/docs/10/static/libpq-envars.html).  Is there a reason we need to invent a new environment variable here?  My bias is to nuke the `DefaultFunc` and provide no default value here.